Diagnostics/Get-NetView.PS1

```diff
@@ -227,138 +227,138 @@ function NetIpNic {
-                            "Get-NetNeighbor | Get-Member"
-        ForEach($cmd in $cmds) {
-            ExecCommand -Command ($cmd) -Output $out
-        }
+    $file = "Get-NetIpAddress.txt"
+    $out  = (Join-Path -Path $dir -ChildPath $file)
+    [String []] $cmds = "Get-NetIpAddress | Format-Table -AutoSize | Out-String -Width $columns",
+                        "Get-NetIpAddress | Format-Table -Property * -AutoSize | Out-String -Width $columns",
+                        "Get-NetIpAddress | Format-List",
+                        "Get-NetIpAddress | Format-List -Property *",
+                        "Get-NetIpAddress | Get-Member"
+    ForEach($cmd in $cmds) {
+        ExecCommand -Command ($cmd) -Output $out
+    }
 
-        $file = "Get-NetIPv4Protocol.txt"
-        $out  = (Join-Path -Path $dir -ChildPath $file)
-        [String []] $cmds = "Get-NetIPv4Protocol | Out-String -Width $columns",
-                            "Get-NetIPv4Protocol | Format-List  -Property *",
-                            "Get-NetIPv4Protocol | Format-Table -Property * -AutoSize",
-                            "Get-NetIPv4Protocol | Format-Table -Property * -AutoSize | Out-String -Width $columns",
-                            "Get-NetIPv4Protocol | Get-Member"
-        ForEach($cmd in $cmds) {
-            ExecCommand -Command ($cmd) -Output $out
-        }
+    $file = "Get-NetIPInterface.txt"
+    $out  = (Join-Path -Path $dir -ChildPath $file)
+    [String []] $cmds = "Get-NetIPInterface | Out-String -Width $columns",
+                        "Get-NetIPInterface | Format-Table -AutoSize  | Out-String -Width $columns",
+                        "Get-NetIPInterface | Format-Table -Property * -AutoSize | Out-String -Width $columns",
+                        "Get-NetIPInterface | Get-Member"
+    ForEach($cmd in $cmds) {
+        ExecCommand -Command ($cmd) -Output $out
+    }
 
-        $file = "Get-NetIPv6Protocol.txt"
-        $out  = (Join-Path -Path $dir -ChildPath $file)
-        [String []] $cmds = "Get-NetIPv6Protocol | Out-String -Width $columns",
-                            "Get-NetIPv6Protocol | Format-List  -Property *",
-                            "Get-NetIPv6Protocol | Format-Table -Property * -AutoSize",
-                            "Get-NetIPv6Protocol | Format-Table -Property * -AutoSize | Out-String -Width $columns",
-                            "Get-NetIPv6Protocol | Get-Member"
-        ForEach($cmd in $cmds) {
-            ExecCommand -Command ($cmd) -Output $out
-        }
+    $file = "Get-NetNeighbor.txt"
+    $out  = (Join-Path -Path $dir -ChildPath $file)
+    [String []] $cmds = "Get-NetNeighbor | Format-Table -AutoSize | Out-String -Width $columns",
+                        "Get-NetNeighbor | Format-Table -Property * -AutoSize | Out-String -Width $columns",
+                        "Get-NetNeighbor | Get-Member"
+    ForEach($cmd in $cmds) {
+        ExecCommand -Command ($cmd) -Output $out
+    }
 
```

Comment:
It's just the GitHub diff view going insane. Beyond Compare 4 correctly shows this as only whitespace changes.